src/js/collect.jsx

```diff
@@ -0,0 +1,240 @@
+    );
+  };
+
+  render() {
+    const { projectDetails, currentPlotId, projectPlots } = this.state;
+    const {
+      setShowInfo,
+      myHeight,
+      localeText: { home },
+    } = this.context;
+    const currentPlot = projectPlots.find((p) => p.id === currentPlotId);
+    return (
+      <>
+        {this.state.showModal && <LoadingModal message={home?.loading || "Cargando"} />}
+        <CollectMap
+          boundary={projectDetails.boundary}
+          currentPlot={currentPlot}
+          extraParams={this.state.extraParams}
+          goToPlot={this.goToPlot}
+          mapboxToken={this.props.mapboxToken}
+          myHeight={myHeight}
+          projectPlots={projectPlots}
+        />
+        {home && (
+          <SideBar>
+            {/* Layers */}
+            <MenuItem
+              itemName="layer"
+              onClick={this.togglePanel}
+              selectedItem={this.state.visiblePanel}
+              tooltip={home.layersTooltip}
+            >
+              <ToolPanel title="Placeholder">
+                <div>
+                  <NICFIControl
+                    extraParams={this.state.extraParams}
+                    nicfiLayers={this.state.nicfiLayers}
+                    setParams={this.setParams}
+                  />
+                  {currentPlot?.geom && (
+                    <a
+                      download={
+                        "comimo_projectId-" +
+                        this.props.projectId +
+                        "_plotId-" +
+                        this.state.currentPlotId +
+                        ".kml"
+                      }
+                      href={
+                        "data:earth.kml+xml application/vnd.google-earth.kmz, " +
+                        encodeURIComponent(this.geomToKML(currentPlot?.geom))
+                      }
+                    >
+                      Download Plot KML
+                    </a>
+                  )}
+                </div>
+              </ToolPanel>
+            </MenuItem>
+            <SideIcon
+              clickHandler={() => setShowInfo(true)}
+              icon="info"
+              parentClass="disclaimer"
+              tooltip={home.appInfoTooltip}
+            />
+          </SideBar>
+        )}
+        <NavBar
+          currentPlotId={this.state.currentPlotId}
+          goToPlot={this.goToPlot}
+          nextPlot={this.nextPlot}
+          prevPlot={this.prevPlot}
+          setPlotAnswer={this.setPlotAnswer}
+          shiftPlotId={projectPlots[0]?.id}
+        />
+      </>
+    );
+  }
+}
+CollectContent.contextType = MainContext;
+
+export function pageInit(args) {
+  ReactDOM.render(
+    <PageLayout
+      role={args.role}
+      userLang={args.userLang}
+      username={args.username}
+      version={args.version}
+    >
+      <CollectContent mapboxToken={args.mapboxToken} projectId={parseInt(args.projectId || 0)} />
+    </PageLayout>,
+    document.getElementById("main-container")
+  );
+}
```

Comment:
Why is this file brand new? Will this conflict with the recent changes we made to the collect page (#50)?

Reply:
We had to restructure how the top entry files worked, where everything is in a file (component) and those that needed specific components to work like collect and home they live in same directory in folder with same name. (like a library). As for jsx so that bundler knows about them that these are JSX files. Those files as they stand (collect.jsx, home.jsx) don't contain the latest changes we made, and doesn't look like they will be merged. We can update them manually which I did now. I'm not sure about new changes though (pending merge).